docker/runtime/bash/http-trigger/kubeless-shell.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+from subprocess import call
+import bottle
+import os
+import prometheus_client as prom
+
+app = application = bottle.app()
+
+cmdline = '/kubeless/' + os.getenv('MOD_NAME') + '.sh'
+cmdlineprep = 'chmod -R 755 /kubeless'
+
+def func(osparam):
+    global cmdline
+    if osparam:
+	cmdline = cmdline + ' ' + osparam
+    call(cmdlineprep, shell=True)
+    call(cmdline, shell=True)
+
+func_calls = prom.Counter('function_calls_total',
+                           'Number of calls to user function',
+                          ['method'])
+func_errors = prom.Counter('function_failures_total',
+                           'Number of exceptions in user function',
+                           ['method'])
+func_hist = prom.Histogram('function_duration_seconds',
+                           'Duration of user function in seconds',
+                           ['method'])
+
+
+@app.route('/', method=['GET', 'POST'])
+def handler():
+    req = bottle.request
+    method = req.method
+    func_calls.labels(method).inc()
+    print str(bottle.request)
+    with func_errors.labels(method).count_exceptions():
```

Comment:
This `count_exception()` only works if the function communicates function errors by raising exceptions.  So, we should do that thing, or use different code to turn non-zero exit into a `func_errors` increment.

Reply:
if we use `check_call` it will raise an exception if the command returns != 0